lib/src/mount.rs

```diff
@@ -57,3 +57,19 @@ pub(crate) fn mount(dev: &str, target: &Utf8Path) -> Result<()> {
         [dev, target.as_str()],
     )
 }
+
+// Ensure the `/dev` directory exists.
+fn ensure_mountpoint(target: &str) -> Result<()> {
+    std::fs::create_dir_all(target)?;
+    Ok(())
```

Comment:
I'm not totally sure this function carries its weight over simply inlining the `std::fs::create_dir_all(target)` into callers?